programmers/문자열-내-p와-y의-갯수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var answer = true;
+    var pcount = 0;
+    var ycount = 0;
+    for (var i in s){
+        if (s[i] == 'p' || s[i] == 'P'){
+            pcount++;
+        }
+        else if (s[i] == 'y' || s[i] == 'Y'){
+            ycount++;
```

Comment:
if다음에 공백 후 ( ) 는 잘해주셨는데  ( ) 이후에도 공백 넣고 { } 가 나오면 좋을 것 같아용
```suggestion
        if (s[i] == 'p' || s[i] == 'P') {
            pcount++;
        }
        else if (s[i] == 'y' || s[i] == 'Y') {
            ycount++;
        }
```